elf/io/extensions.py

```diff
@@ -72,6 +73,12 @@ def register_filetype(constructor, extensions=(), groups=(), datasets=()):
 except ImportError:
     mrcfile = None
 
+# add bossdb extensions if we have intern
+try:
+    import intern
+    register_filetype(InternFile, ["bossdb://"], InternFile, InternDataset)
+except ImportError:
+    pass
```

Comment:
Is there a reason you were doing `mrcfile = None` in the other extensions? i.e. do you want me to add `intern = None`?

Reply:
Yes, this is used to raise a more meaningful error message [here](https://github.com/constantinpape/elf/blob/master/elf/io/mrc_wrapper.py#L48-L49).
(Although the current error message for mrc is not very helpful; I think we should add installation hints for intern and eventually for mrc as well.)

Reply:
Just added a hint!